packages/veritone-widgets/src/widgets/EngineSelection/EngineListView/EngineSelectionRow/index.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { bool, object, func, string, shape } from 'prop-types';
+
+import { Lozenge, Ellipsis } from 'veritone-react-common';
+
+import { modules } from 'veritone-redux-common';
+const { engine: engineModule } = modules;
+
+import Checkbox from 'material-ui/Checkbox';
+
+import networkIsolatedLogo from '../../images/networkisolated_logo.png';
+import externalAccessLogo from '../../images/externalaccess_logo.png';
+import externalProcessingLogo from '../../images/externalprocessing_logo.png';
+import humanReviewLogo from '../../images/humanreview_logo.png';
+
+import * as engineSelectionModule from '../../../../redux/modules/engineSelection';
+
+import ToggleButton from '../../ToggleButton/';
+
+import styles from './styles.scss';
+
+@connect(
+  (state, ownProps) => ({
+    isSelected: engineSelectionModule.engineIsSelected(
+      state,
+      ownProps.engineId
+    ),
+    engine: engineModule.getEngine(state, ownProps.engineId),
+    isChecked: engineSelectionModule.engineIsChecked(state, ownProps.engineId)
+  }),
+  {
+    addEngines: engineSelectionModule.addEngines,
+    removeEngines: engineSelectionModule.removeEngines,
+    checkEngine: engineSelectionModule.checkEngine,
+    uncheckEngine: engineSelectionModule.uncheckEngine
+  }
+)
+export default class EngineSelectionRow extends React.Component {
+  static propTypes = {
+    engine: shape({
+      id: string.isRequired,
+      name: string.isRequired,
+      category: object,
+      description: string,
+      iconPath: string,
+      ownerOrganization: object
+    }).isRequired,
+    isSelected: bool.isRequired,
+    isChecked: bool.isRequired,
+    onViewDetail: func.isRequired,
+    addEngines: func.isRequired,
+    removeEngines: func.isRequired,
+    checkEngine: func.isRequired,
+    uncheckEngine: func.isRequired
+  };
+
+  handleOnChange = () => {
```

Comment:
this is just a stylistic issue, but these should be `handleChange`, `handleClick` (saw this issue in a few places). Here's the eslint rule to prove i'm not just making this up: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-handler-names.md (maybe we should add it)